project/jteng2127/README.rst

```diff
@@ -0,0 +1,199 @@
+##########
+ PyBezier
+##########
+
+*******************
+ Basic Information
+*******************
+
+**Project**: pybezier
+
+**Github url**: https://github.com/jteng2127/pybezier
+
+******************
+ Problem to Solve
+******************
+
+In the fields of Aerodynamics, Fluid mechanics, and Modeling, it's often
+essential to describe complex curves accurately. One approach involves
+the use of Bezier curves, which are widely applied in computer graphics
+because their smoothness and intuitive control.
+
+The goal is to use multiple third-order (or higher order) Bezier curves
+to fit the following types of curve:
+
+#. A fourth-order Bezier curve
+#. A NACA 4-digit airfoil [1]_
+#. Any arbitrary curve
```

Comment:
What will be the mathematic form of any arbitrary curve you are expecting to handle?

Reply:
I've updated some example for it.

Reply:
Don't resolve inline comments.  It's harder to track.